README.md

```diff
@@ -66,12 +66,15 @@ Usage:  <br>
 |`--ont FILE ...` | validate against the OWL-formatted ontolog(ies) at the specified filename(s) |
 |`--nist` | validate against the NIST restrictions |
 |`--nist-ta3` | validate against the NIST hypothesis restrictions (implies `--nist`) |
-|`-o` | Save validation report model to a file.  `KB.ttl` would result in `KB-report.txt`. Output defaults to stderr. |
-|`-h, --help` | This help and usage text |
 |`--abort [num]` | Abort validation after `[num]` SHACL violations (num > 2), or three violations if `[num]` is omitted. |
-|`--pm | Enable progress monitor that shows ongoing validation progress |
-|`-f FILE ...` | validate the specified file(s) with a `.ttl` suffix |
+|`--pm` | Enable progress monitor that shows ongoing validation progress |
+|`-o` | Save validation report model to a file.  `KB.ttl` would result in `KB-report.txt`. Output defaults to stderr. |
+|`-t num` | Specify the number of threads to use during validation |
 |`-d DIRNAME` | validate all `.ttl` files in the specified directory |
+|`-f FILE ...` | validate the specified file(s) with a `.ttl` suffix |
+|`-h, --help` | This help and usage text |
+|`-h, --help` | This help and usage text |
```

Comment:
Extra `-h` line, and could you add the equals sign to `-f`, `-d`, and `-t` options?

Reply:
fixed